[2024-08-11T03:27:56.572+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-11T03:27:56.591+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.create_bucket scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-08-11T03:27:56.597+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.create_bucket scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-08-11T03:27:56.597+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-11T03:27:56.614+0000] {taskinstance.py:2330} INFO - Executing <Task(GCSCreateBucketOperator): create_bucket> on 2024-01-01 00:00:00+00:00
[2024-08-11T03:27:56.622+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=61) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-11T03:27:56.624+0000] {standard_task_runner.py:64} INFO - Started process 71 to run task
[2024-08-11T03:27:56.624+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'create_bucket', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/googleBigQuery.py', '--cfg-path', '/tmp/tmpzlz_zbyy']
[2024-08-11T03:27:56.626+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask create_bucket
[2024-08-11T03:27:56.680+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag.create_bucket scheduled__2024-01-01T00:00:00+00:00 [running]> on host b6e1331e9d6b
[2024-08-11T03:27:56.767+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='create_bucket' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-08-11T03:27:56.769+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-11T03:27:56.786+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-08-11T03:27:56.787+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-08-11T03:27:56.928+0000] {gcs.py:1080} INFO - Creating Bucket: leptospirosis_bucket; Location: US; Storage Class: MULTI_REGIONAL
[2024-08-11T03:27:57.869+0000] {gcs.py:161} WARNING - Bucket leptospirosis_bucket already exists
[2024-08-11T03:27:57.870+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-11T03:27:58.060+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=create_bucket, run_id=scheduled__2024-01-01T00:00:00+00:00, execution_date=20240101T000000, start_date=20240811T032756, end_date=20240811T032758
[2024-08-11T03:27:58.131+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-11T03:27:58.156+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-11T03:27:58.163+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
